.github/scripts/attack.py

```
@@ -0,0 +1,341 @@
+    Key: disconnect quickly each time so X returns to cache=1 most of the time.
+    """
+    log = mk_logger("FLAP")
+    i = 0
+    while not stop.is_set():
+        clean_start = (i % 5 == 4)  # 4x resume + 1x takeover
+        i += 1
+
+        ev = threading.Event()
+        c = new_client(CLIENTID_X)
+
+        def on_connect(cl, userdata, flags, reason_code, properties):
```

Ruff (0.14.10)

Unused function argument: `cl`

(ARG001)